### PR TITLE
BUGFIX RedirectorPage::Link() results in unexpected disallows

### LINKED ISSUE #14: BUG RedirectorPage set to not show in search could result in destination page link being disallowed
We've found some cases where a shorter url is setup with a redirector page, and the redirect to url is actually disallowed as a result. I feel like this is an edge case based on the link that's returned by RedirectorPage, however it causes some indexing issues from an SEO perspective.

There is the config option to allow url's, however this seems inefficient as a code update would be required when a redirector is added to the cms. Curious on thoughts for this as for the time being we're having the client turn on the "Show in search" option for those redirector pages as that seems to be fine for a solution.

### PR DESCRIPTION
**Pull request by muskie9**
resolves #14

### CODE REVIEW

File: src/Robots.php [hunk 1]
```diff
@@ -140,7 +140,7 @@ protected function disallowedUrls()
     {
         // If not public, disallow all
         if (!$this->isPublic()) {
-            return array("/");
+            return ["/"];
```
Comment (muskie9) [inline src/Robots.php:143]: I think the IDE changed this on accident.

Comment (tractorcow) [reply src/Robots.php:143]: All good bro

**Comment by tractorcow** [comment]
Thanks @muskie9 for jumping onto this.

**Comment by tractorcow** [comment]
Added 4.0.2 tag

**Comment by muskie9** [comment]
Thanks @tractorcow

**Comment by muskie9** [comment]
I was debating adding tests, but noticed the base query references `SiteTree` but the module requires `framework`. I could see this module being useful for non-cms use cases, but the query seems to couple it fairly tightly. Is it worth updating the composer requirement (not to drop a new issue in the comments lol)?

**Pull request merged**